subsys/dfu/boot/targets/96b_nitrogen.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2017 Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Bootloader device specific configuration.
+ */
+
+#define FLASH_DRIVER_NAME		CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_ALIGN			4
+#define FLASH_AREA_IMAGE_0_OFFSET	0x08000
```

Comment:
To confirm, we need to solve the driver name issue. I think a future solution will be to have this defined entirely by the device tree (and not even using a Kconfig option). But, it seems clumsy to add a bunch of target-specific headers that each contain a single #define of a device name, but I'm not sure how else to handle it until the other aspect is solved. @agross-linaro thoughts?